src/utils/beneficial.owner.statements.ts

```diff
@@ -8,26 +8,40 @@ import { checkBOsDetailsEntered, checkMOsDetailsEntered, getApplicationData, set
 import { BeneficialOwnersStatementType, BeneficialOwnerStatementKey } from "../model/beneficial.owner.statement.model";
 import { saveAndContinue } from "../utils/save.and.continue";
 
-export const getBeneficialOwnerStatements = (req: Request, res: Response, next: NextFunction, registrationFlag: boolean) => {
+export const getBeneficialOwnerStatements = (req: Request, res: Response, next: NextFunction, registrationFlag: boolean, noChangeBackLink?: string) => {
   try {
-    const BACK_LINK = registrationFlag ? config.ENTITY_URL : config.OVERSEAS_ENTITY_REVIEW_URL;
+    let BACK_LINK: string;
+
+    if (noChangeBackLink){
+      BACK_LINK = noChangeBackLink;
+    } else {
+      BACK_LINK = registrationFlag ? config.ENTITY_URL : config.OVERSEAS_ENTITY_REVIEW_URL;
+    }
+
     logger.debugRequest(req, `${req.method} ${config.BENEFICIAL_OWNER_STATEMENTS_PAGE}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
     return res.render(config.BENEFICIAL_OWNER_STATEMENTS_PAGE, {
       backLinkUrl: BACK_LINK,
       templateName: config.BENEFICIAL_OWNER_STATEMENTS_PAGE,
-      [BeneficialOwnerStatementKey]: appData[BeneficialOwnerStatementKey]
+      [BeneficialOwnerStatementKey]: appData[BeneficialOwnerStatementKey],
+      noChangeFlag: true
```

Comment:
Is this flag not always going to be true for any controller that utilises this method? (Same goes for src/utils/registrable.beneficial.owner.ts) So the page will always be rendered in the noChange scenario

Reply:
@bbell-companieshouse good spot. Fixed